models/models.js

```diff
@@ -1,12 +1,112 @@
-var mongoose = require("mongoose");
-var userSchema = new mongoose.Schema({
-    username: String,
-    email: String,
-    password: String,
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const bcrypt = require('bcrypt-nodejs');
+
+
+
+let usernameLengthChecker = (username) => {
+    if (!username) {
+        return false;
+    }
+    if (username.length < 3 || username.length > 15) {
```

Comment:
Do not use magic numbers, create constants one line above - MIN_LENGTH and MAX_LENGTH